ui/components/multichain/token-list-item/token-list-item.js

```diff
@@ -27,11 +27,11 @@ import {
   IconName,
   IconSize,
   Modal,
-  ModalContent,
-  ModalHeader,
   ModalOverlay,
   Text,
 } from '../../component-library';
+import { ModalContent } from '../../component-library/modal-content/deprecated';
+import { ModalHeader } from '../../component-library/modal-header/deprecated';
 import {
   getCurrentChainId,
   getCurrentNetwork,
```

Comment:
Had to comment out some code to show modal as there was no effective way to do it via storybook so only have the After screenshot

### After

<img width="1510" alt="Screenshot 2023-12-10 at 5 00 06 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/fdbbf4d0-35bb-459d-9c88-f009fe6715ed">